.github/workflows/carthage.yml

```diff
@@ -0,0 +1,24 @@
+name: Carthage
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
```

Comment:
Is this comment correct, below it looks like we triggering on both master & develop?
If so can remove from all the other PR's

Reply:
You're right, fixed here and in the related MRs for other projects